mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -546,6 +546,48 @@ func schemaNodeConfig() *schema.Schema {
 						},
 					},
 				},
+				"sole_tenant_config": {
+					Type:        schema.TypeList,
+					Optional:    true,
+					Computed:    true,
```

Comment:
Does the GKE API return a default object when `sole_tenant_config` is unset? I don't see that happening on preexisting test configs. If it doesn't, we can omit `Computed` here.